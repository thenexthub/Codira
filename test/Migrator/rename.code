// REQUIRES: objc_interop
// RUN: %empty-directory(%t) && %target-language-frontend -c -update-code -primary-file %s -F %S/mock-sdk -api-diff-data-file %S/Inputs/API.json -emit-migrated-file-path %t/rename.code.result -emit-remap-file-path %t/rename.code.remap -o /dev/null -disable-migrator-fixits
// RUN: diff -u %S/rename.code.expected %t/rename.code.result

import Bar

fn foo(_ b: BarForwardDeclaredClass) {
  b.barInstanceFunc1(0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  _ = barGlobalVariableOldEnumElement
  _ = PropertyUserInterface.methodPlus()
  immutable _: BarBaseNested
  _ = barGlobalHoistedFuncOldName(0, 1, 2)
}

fn foo1(_ b: BarForwardDeclaredClass) {
  b.barInstanceFunc2(0,toRemove:1,toRemove1:2,toRemove2:3)
}
