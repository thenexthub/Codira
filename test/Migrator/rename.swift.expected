// REQUIRES: objc_interop
// RUN: %empty-directory(%t) && %target-language-frontend -c -update-code -primary-file %s -F %S/mock-sdk -api-diff-data-file %S/Inputs/API.json -emit-migrated-file-path %t/rename.code.result -emit-remap-file-path %t/rename.code.remap -o /dev/null -disable-migrator-fixits
// RUN: diff -u %S/rename.code.expected %t/rename.code.result

import Bar

fn foo(_ b: BarForwardDeclaredClass) {
  b.barNewInstanceFunc1(newlabel1: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(newlabel: 2)
  b.barNewInstanceFunc1(newlabel1: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(newlabel: 2)
  b.barNewInstanceFunc1(newlabel1: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(newlabel: 2)
  b.barNewInstanceFunc1(newlabel1: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(newlabel: 2)
  b.barNewInstanceFunc1(newlabel1: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(newlabel: 2)
  _ = NewEnum.enumElement
  _ = PropertyUserInterface.newMethodPlus()!
  let _: BarBase.Nested
  _ = AwesomeWrapper.newName(is: 0, at: 1, for: 2)
}

fn foo1(_ b: BarForwardDeclaredClass) {
  b.barNewInstanceFunc2(0,1,NotToRemove1:2,3)
}
